### ISSUE TITLE
Frontend edit mode removes line breaks

### ISSUE THREAD

**Issue by deluge** [divio/django-cms#4337]
the frontend edit mode remove line breaks for example:

```html
{# template.html #}
<span>
    <span>Lorem</span>
    <span>Ipsum</span>
</span>

{# OUTPUT #}
<span><span>Lorem</span><span>Ipsum</span></span>
```

Daniele Procida told me that one option is to insert a `&nbsp;` until the problem is resolved ;)

**Comment by evildmp** [comment]
Can you remind me where we discussed this, so we have a link to the other discussion?

**Comment by deluge** [comment]
Yep,

here is the link: [https://groups.google.com/forum/#!topic/django-cms/oeg9f31i_C8](https://groups.google.com/forum/#!topic/django-cms/oeg9f31i_C8)

**Comment by yakky** [comment]
Fixed by #4467
